portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,135 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="intro-wrapper">
+      <div id="intro">
+        Hello world, I am
+        <span id="name">Bao Xian</span>
+      </div>
+    </div>
+    
+    <div id="content" class='hide'>
+
+      <div id='main' class='section'>
+        <h1>My name is Bao Xian
+          <span id='dp'></span>
+        </h1>
+        <p>
+          I'm an Information Systems student from Singapore, aspiring to be a software developer / product manager.
+        </p>
+        <p>
+          This is my personal portfolio.
+        </p>
+        <p>
+          <button onclick='randomFact()'>Click here</button> for a random fact about me!<br/>
+          <span id='fact'></span>
+        </p>
+        <p>
+          Check out my other profiles in the buttons just below!
+        </p>
+        
+        <a class='logo' id='linkedin' href='https://www.linkedin.com/in/baoxiantsang/'></a>
+        <a class='logo' id='github' href='https://github.com/bxtsang'></a>
+        <a class='logo' id='instagram' href='https://www.instagram.com/baoxiannn/'></a>
+        <!-- <a class='logo' id='next' href="#about"></a> -->
+      </div>
+
+      <div id="about" class='section'>
+        <h1>More about me...</h1>
+        <h3>Water Polo</h3>
+        <p>
+          I am a huge water polo lover. I've played the sport competitively since I was 14 in
+          my various school teams, been part of the <b>Singapore National Men's Water Polo team</b> from 2016 to 2018, and captained the SMU team.
+          I've also been a coach for many youth and school teams, coaching players between 9 to 17 years old.
+        </p>
+        <p>
+          Water polo has moulded my character, teaching me the very basics of discipline and hardwork,
+          while also training me to work well in teams using good communication and leadership skills.
+          Hence, even as I pursue a profession now away from this sport, the mindset I've inherited from water polo still very much defines me.
+        </p>
+        <button onclick="toggleImages()">See some pictures!</button><br/>
+        <img src="images/wp2.jpg" class='wp-img hide' />
```

Comment:
not important but, it is usually not a good idea to name assets file into something that is easy to predict, like <filename>1-3
if you decided to not use <filename>2 for now people may be able to guess your file name

some suggestion is to either use UUID as the file name, or, put the image file into UUID named folder
for example:
images/5a251312-28a9-4584-ad32-75251348499b/wp2.jpg

Reply:
Oh, will they be unable to access the files with UUID? Or what are the dangers of them guessing the file names?

Reply:
Basically it is a good practice to not allow people to access file which is not used in your website by guessing. since the assets stored in your website is publicly accessible, the path name to that file is basically the password to access it, and its usually better to make it as hard to guess as possible

Reply:
Ok got it! Will set up a folder with UUID as I add more images